docs/release-channels.md

```diff
@@ -5,13 +5,13 @@ Microk8s is a snap deploying Kubernetes. Upstream Kubernetes ships a new release
 
 ## Choosing the Right Channel
 
-When installing microk8s you can select your desired upstream Kubernetes series with the corresponding snap channel. For example, to install microk8s and let it follow the `v1.12` release series you:
+When installing MicroK8s you can select your desired upstream Kubernetes series with the corresponding snap channel. For example, to install KicroK8s and let it follow the `v1.12` release series you:
```

Comment:
KicroK8s!

Reply:
Unacceptable! Both the typo and the review! You are on vacations, we do not accept reviews from you @tvansteenburgh